stable/rabbitmq/templates/configuration.yaml

```diff
@@ -24,6 +24,19 @@ data:
     ssl_options.certfile = /opt/bitnami/rabbitmq/certs/server_certificate.pem
     ssl_options.keyfile = /opt/bitnami/rabbitmq/certs/server_key.pem
 {{- end }}
+{{- if .Values.ldap.enabled }}
+    auth_backends.1 = rabbit_auth_backend_ldap
+    auth_ldap.servers.1  = {{ .Values.ldap.server }}
+    auth_ldap.port = {{ .Values.ldap.port }}
+    auth_ldap.user_dn_pattern = {{ .Values.ldap.user_dn_pattern  }}
```

Comment:
I'd add a validation (validateValues macro) to ensure that every LDAP parameter is set when LDAP is enabled.